conf/development.ini

```diff
@@ -0,0 +1,30 @@
+[app:main]
```

Comment:
Bouncer didn't have a `development.ini` file. I just copy-pasted the one from `test-pyramid-app`